lib/Fmt_odoc.ml

```diff
@@ -89,7 +89,8 @@ let fmt_code_block conf s1 s2 =
       let box = match lines with _ :: _ :: _ -> vbox 0 | _ -> hvbox 0 in
       box (wrap_code (vbox 0 (list_fl lines fmt_line)))
 
-let fmt_code_span s = hovbox 0 (wrap "[" "]" (str (escape_brackets s)))
+let fmt_code_span s =
+  hovbox 0 (wrap "[" "]" (str_normalized ~escape:escape_brackets s))
```

Comment:
`str_normalized` inserts a break between each word. UTF-8 length calculation is also done by `str`.
Code span were lacking breaks on purpose because they are expected to contain a small amount of text but I'm fine with adding breaks.

Reply:
You're right about length calculation. The issue is missing breaks, which get added with this patch.